vent/controller/control_module.py

```diff
@@ -443,30 +441,35 @@ def run(self):
         # start running
```

Comment:
When is `run()` called? I didn't see the class call it.

Reply:
run()? This is done by python's threading after start()

Here is the ref: "Once a thread object is created, its activity must be started by calling the thread’s start() method. This invokes the run() method in a separate thread of control... Once the thread’s activity is started, the thread is considered ‘alive’. It stops being alive when its run() method terminates" [https://docs.python.org/3/library/threading.html ]

Reply:
I see. I didn't know this way of inherit a thread. Thanks for letting me knowing this!

Nits: this way seems better in cases where you are creating a thread class (loading threads). But for our current controller class, not sure this is good because later we may have multiple threads inside this class.

Reply:
Yeah, this is not relevant anymore with the revised design